refactor(ipfs): endpoint and flag Scaleway CLI Labs (#1756)
scaleway/docs-content: labs/ipfs/api-cli/ipfs-operations.mdx

  It suppose that the content pointed by the CID must already be present on the public IPFS network.
</Message>

<Message type="important">

>     To use the ipfs command in the Scaleway CLI, set the SCW_ENABLE_LABS=true flag.

Reviewer: ```suggestion
    To manage IPFS with the Scaleway CLI, you must set the `SCW_ENABLE_LABS` flag to `true`, as such: `SCW_ENABLE_LABS=true`
```

>     It's advisable to export it as an environment variable, so you don't have to set it for every command.

Reviewer: ```suggestion
    We recommend you export the flag as an environment variable, so you do not have to set it for every command.
```
